a10_octavia/controller/worker/flows/a10_member_flows.py

```diff
@@ -827,6 +831,26 @@ def get_rack_vthunder_batch_update_members_flow(self, old_members, new_members,
         batch_update_members_flow.add(server_tasks.MemberFindNatPool(
             requires=[a10constants.VTHUNDER, constants.POOL, constants.FLAVOR],
             provides=a10constants.NAT_FLAVOR))
+        members = old_members + new_members
+        batch_update_members_flow.add(a10_database_tasks.CountBatchMembersWithIPPortProtocol(
+            inject={a10constants.MEMBERS: members},
+            requires=constants.POOL,
+            provides=(a10constants.MEMBER_COUNT_IP_DICT,
+                      a10constants.MEMBER_COUNT_IP_PORT_PROTOCOL_DICT)))
+        batch_update_members_flow.add(a10_database_tasks.GetNatPoolEntryForBatchMembers(
+            inject={a10constants.MEMBERS: members},
+            requires=[a10constants.NAT_FLAVOR],
+            provides=a10constants.NAT_POOL_DICT))
+        if new_members:
```

Comment:
Can we move the address release/reservation after member are created/deleted?
In case error happened, we don't need to implement reverse function for them.